packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -60,6 +60,10 @@ export const PaymentConfirmation = ({
   );
 
   const { l10n } = useLocalization();
+  const downloadLink: string =
+    productUrl.indexOf('?') > -1
+      ? productUrl + `&email=${encodeURIComponent(email)}`
+      : productUrl + `?email=${encodeURIComponent(email)}`;
```

Comment:
I think this probably would work for most cases.  But a safer way to do this is to use the URL API: https://developer.mozilla.org/en-US/docs/Web/API/URL.  Once the url's parsed you can add an entry into its search params (https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams).

Reply:
updated to use URL API